test/backend/test_dynamic_shape.py

```diff
@@ -3,17 +3,17 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-import numpy as np
-import tensorflow as tf
-import unittest
-
 from onnx_tf.backend import onnx_graph_to_tensorflow_rep
 from onnx_tf.common.legacy import legacy_opset_pre_ver
 from onnx_tf.common.pooling_helper import py_pool
 from onnx import defs
 from onnx import helper
 from onnx import TensorProto
 
+import numpy as np
+import tensorflow as tf
+import unittest
+
```

Comment:
We have recommended to follow the import order, with 3 main sections:
1. system imports
2. software dependency imports, ie onnx and tensorflow
3. own packages, ie onnx_tf

Examples can be found in backend.py, converter.py, etc.

As you found here, not all files follow the recommend import order. I would like suggest we keep the same 3 sections whenever possible. So if needed, we just need to ensure onnx import comes before tensorflow in the middle sections.

Reply:
The changes look good! Can you please resolve the branch conflicts?